docs/querying/sql-data-types.md

```diff
@@ -33,7 +33,7 @@ Columns in Druid are associated with a specific data type. This topic describes
 
 Druid natively supports five basic column types: "long" (64 bit signed int), "float" (32 bit float), "double" (64 bit
 float) "string" (UTF-8 encoded strings and string arrays), and "complex" (catch-all for more exotic data types like
```

Comment:
```suggestion
Druid natively supports the following basic column types: "long" (64-bit signed int), "float" (32-bit float), "double" (64-bit
float) "string" (UTF-8 encoded strings and string arrays), "complex" (catch-all for more exotic data types like
```

Reply:
```suggestion
Druid natively supports the following column types: "long" (64-bit signed int), "float" (32-bit float), "double" (64-bit
float) "string" (UTF-8 encoded strings and string arrays), and "complex" (non-standard data types, such as nested JSON, hyperUnique and approxHistogram aggregators, and DataSketches aggregators)
```

Reply:
I like the list formatting for readability; don't forget to remove L36 if using this option